andela/aragon-cfh
# Feature/141404599/CZAR should be able to draw cards by clicking

#### What does this PR do?
Implement deck of black cards of the next question card that the CZAR should be prompted to click.

#### Description of Task to be completed?
A modal/dialog shows up when the user is assigned as the CZAR, prompting them to click the deck of cards.
There is a deck of black cards. When a user clicks on the deck, the next card transition in, by flipping and consequently reveals the next question.

#### How should this be manually tested?
Run the app and open it in a browser.
The deck of black cards appears on the CZAR's board on clicking start game. The remaining players wait until the CZAR clicks on the deck of cards before the game starts. 

If after the expiration of time the CZAR did not click on the deck of cards then the CZAR authority shifts to another player next in line.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
